rfcs/0007-filtering.md

```diff
@@ -0,0 +1,127 @@
+# RFC Proposal
+[RFC Proposal]: #rfc-proposal
+
+* RFC Proposal ID: `0007-filtering-fields`
+* Start Date: 2020-08-21
+* RFC PR: [rfcs#0007](https://github.com/CMSgov/beneficiary-fhir-data/pull/345)
+* JIRA Ticket(s):
+    * [https://jira.cms.gov/browse/AB2D-1863](https://jira.cms.gov/browse/AB2D-1863)
+
+This proposal will suggest adding server side filtering to API calls that return EOBs to the client in order to 
+cut down on the amount of data that needs to be transferred. 
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+* [RFC Proposal](#rfc-proposal)
+* [Table of Contents](#table-of-contents)
+* [Motivation](#motivation)
+* [Proposed Solution](#proposed-solution)
+    * [Proposed Solution: Detailed Design](#proposed-solution-detailed-design)
+    * [Proposed Solution: Unresolved Questions](#proposed-solution-unresolved-questions)
+    * [Proposed Solution: Drawbacks](#proposed-solution-drawbacks)
+    * [Proposed Solution: Notable Alternatives](#proposed-solution-notable-alternatives)
+* [Future Possibilities](#future-possibilities)
+
+## Motivation
+[Motivation]: #motivation
+
+AB2D receives lots of data back from BFD when we make API calls to gather EOBs (Explanation of Benefits), 
+where we filter out much of the data we receive. This might not be terrible for a few API calls, but for 
```

Comment:
> where we filter out much of the data we receive

Can you be explicit about what % of the fields would be filtered out? I think that might help reduce confusion.

Reply:
To be clear: doesn't have to be exact -- a rough estimate would be great.

Reply:
Rough estimate:  I counted getters in org.hl7.fhir.dstu3.model.ExplanationOfBenefit

69 top level getters
10 used by AB2D

ExplanationOfBenefit encapsulates ItemComponent which has

42 top level getters
 6 used by AB2D